app/components/sidebars/main/channels_list/channels_list.js

```diff
@@ -48,6 +49,12 @@ export default class ChannelsList extends PureComponent {
         });
     }
 
+    componentWillReceiveProps(nextProps) {
+        if (!nextProps.drawerOpened && (this.props.drawerOpened !== nextProps.drawerOpened)) {
```

Comment:
Perhaps I'm missing something, but this seems like it would be more clearly expressed as:
```
if (!nextProps.drawerOpened && this.props.drawerOpened) {
   // ...
}
```

Reply:
No, you are not. That is definitely better. Thanks